### PR TITLE
Move Media APIs to the new components and clean up unnecessary stuff from Pages and Pages.Config

### PR DESCRIPTION
**Pull request by MarcinJuraszek**

### CODE REVIEW

**Comment by erinha** [comment]
<p>Analyzed commit id : 24ca2afcc7337decaebc1e0d1617a4c6cf2815b2</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: c6b6c6c6efecced624c83faa0f1b708bef9824cb</p>

File: apps/teams-test-app/src/components/MediaAPIs.tsx [hunk 1]
```diff
@@ -101,12 +108,52 @@ const MediaAPIs = (): ReactElement => {
...
+    name: 'viewImagesWithUrls2',
+    title: 'View Images With Urls',
+    onClick: {
+      validateInput: input => {
+        if (!input || !Array.isArray(input) || input.length === 0 || input.find(x => typeof x !== 'string')) {
+          throw new Error('input has to be an array of strings with at least one element');
+        }
+      },
+      submit: async input => {
+        const urlList: media.ImageUri[] = input.map(x => ({ value: x, type: 2 }));
```
Comment (aeunms) [inline apps/teams-test-app/src/components/MediaAPIs.tsx:138]: Nit: consider leaving in the comment to show what 2 is? `//ImageUriType.URL`

**Review by aeunms — APPROVED** [review]
LGTM, left one small comment

**Comment by erinha** [comment]
<p>Analyzed commit id : d78a1366b5a33e81917e7baa71f9c3b3acafd759</p><hr><details><summary><b><span style="color: green">⯆</span> @microsoft/bundle-analysis-app:</b> -6 Bytes</summary><table><tr><th>Metric Name</th><th>Baseline Size</th><th>Compare Size</th><th>Size Diff</th></tr><tr><td>main.js</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr><tr><td>Total Size</td><td>58.4 KB</td><td>58.4 KB</td><td><span style="color: green">⯆</span> -6 Bytes</td></tr></table></details><hr><p>Baseline commit: c6b6c6c6efecced624c83faa0f1b708bef9824cb</p>

**Review by aeunms — APPROVED** [review]

**Pull request merged**